kubernetes/opentelemetry-demo.yaml

```diff
@@ -164,15 +43,24 @@ metadata:
   labels:
     app.kubernetes.io/name: otelcol
     app.kubernetes.io/instance: opentelemetry-demo
-    app.kubernetes.io/version: "0.92.0"
+    app.kubernetes.io/version: "0.102.1"
     
 data:
   relay: |
     connectors:
-      spanmetrics: null
+      spanmetrics: {}
     exporters:
       debug: {}
-      logging: {}
+      googlecloud:
+        log:
+          default_log_name: opentelemetry.io/opentelemetry-demo
+      googlemanagedprometheus: null
+      opensearch:
```

Comment:
weird that this still shows up even though it's disabled, I guess it's not in the pipeline though

Reply:
They're hardcoded in the upstream Helm chart. But I was able to get them to go away by overriding them to `null`.